src/pybind/mgr/dashboard/controllers/rbd_mirroring.py

```diff
@@ -507,6 +506,9 @@ def get(self, pool_name):
 
     @RbdMirroringTask('pool/edit', {'pool_name': '{pool_name}'}, 5.0)
     def set(self, pool_name, mirror_mode=None):
+        return self.set_pool_mirror_mode(pool_name, mirror_mode)
```

Comment:
we can move the logic of this function to `services/rbd.py` file. That way we can remove the circular dependency as well. That way we ensure reusability. Its always better to import a service rather than a controller. Let me know if you went into any issues when doing that..

Reply:
@nizamial09 , by moving this method to `services/rbd.py`, we need to move atleast 4 pre-existing methods to rbd.py service and adjust controller's code accordingly and am not sure if such a change is expected from this PR